spec/models/topic_spec.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id          :bigint           not null, primary key
+#  description :string(50)       not null
+#  image       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
```

Comment:
What might be a good thing to test (and easy) are your validations. You can test that the validation on `describe` works fine